packages/checkout-ui-extensions/src/components/Button/README.md

```diff
@@ -7,13 +7,13 @@ optional = ?
 
 | Name | Type | Description |
 | --- | --- | --- |
-| kind? | <code>"primary" &#124; "secondary" &#124; "plain"</code> | The type of button that will be rendered. The visual of these buttons are controlled by merchants through the Branding API.<br /><br /> - `primary`: button used for main actions. Ex: &#34;Continue to next step&#34;<br /><br />- `secondary`: button used for secondary actions not blocking user progress. Ex: &#34;Download Shop app&#34;<br /><br />- `plain`: renders a button that visually looks like a Link<br /><br /> Default value: <code>'primary'</code> |
+| kind? | <code>"primary" &#124; "secondary" &#124; "plain"</code> | The type of button that will be rendered. The visual presentation of the button type is controlled by merchants through the Branding API.<br /><br /> - `primary`: button used for main actions. Ex: &#34;Continue to next step&#34;<br /><br />- `secondary`: button used for secondary actions not blocking user progress. Ex: &#34;Download Shop app&#34;<br /><br />- `plain`: renders a button that visually looks like a Link<br /><br /> Default value: <code>'primary'</code> |
 | appearance? | <code>"monochrome" &#124; "critical"</code> | Specify the color treatment of the Button. By default, it will use the &#34;Primary / Secondary Action&#34; colors.<br /><br /> - `monochrome` will take the color of its parent.<br /><br />- `critical` will take the critical color.  |
 | submit? | <code>boolean</code> | Allows the button to submit a form  |
-| to? | <code>string</code> | Destination to link to, renders a Link  |
-| inlineAlignment? | <code>"leading" &#124; "center" &#124; "trailing"</code> | Specifies the inline alignment of the content.<br /><br />Default value: <code>'center'</code> |
-| loading? | <code>boolean</code> | Replaces content with a loading indicator  |
-| loadingLabel? | <code>string</code> | Accessible label for the loading indicator when user prefers reduced motion  |
+| to? | <code>string</code> | Destination URL to link to. If this value is set, the button will render as a Link.  |
+| inlineAlignment? | <code>"leading" &#124; "center" &#124; "trailing"</code> | Specifies the inline alignment of the content<br /><br />Default value: <code>'center'</code> |
+| loading? | <code>boolean</code> | Replaces content with a loading indicator<br /><br />Default value: <code>false</code> |
+| loadingLabel? | <code>string</code> | Accessible label for the loading indicator when user prefers reduced motion. This value is only used if `loading` is true.  |
```

Comment:
```suggestion
| loadingLabel? | <code>string</code> | Accessible label for the loading indicator when users prefer reduced motion. This value is only used if `loading` is true.  |
```

Reply:
Leaving this out since this content comes from the package source (code comments) and not related to this change.